cypress.config.ts

```diff
@@ -23,57 +20,37 @@ export default defineConfig({
   },
   videoUploadOnPasses: false,
   taskTimeout: 60000,
-  viewportWidth: 2000,
-  viewportHeight: 2000,
+  viewportWidth: 1280,
+  viewportHeight: 1400,
   e2e: {
-    // We've imported your old cypress plugins here.
-    // You may want to clean this up later by importing these.
     setupNodeEvents(on) {
       on('task', {
         reset: resetStubs,
         ...auth,
         ...tokenVerification,
-        stubGetPrisonerSocialContacts: (offenderNo: string) =>
-          prisonerContactRegistry.getPrisonerSocialContacts(offenderNo),
-        stubGetOffenderEvents: ({
-          offenderNo,
-          fromDate,
-          toDate,
-        }: {
-          offenderNo: string
-          fromDate: string
-          toDate: string
-        }) =>
-          whereaboutsOffenderEvents.stubGetOffenderEvents({
-            offenderNo,
-            fromDate,
-            toDate,
-          }),
-        stubGetBookings: (offenderNo: string) => prisonApi.getBookings(offenderNo),
-        stubGetOffender: (prisoner: Partial<InmateDetail>) => prisonApi.getOffender(prisoner),
-        stubGetOffenderRestrictions: (offenderNo: string) => prisonApi.getOffenderRestrictions(offenderNo),
-        stubGetVisitBalances: ({ offenderNo, visitBalances }: { offenderNo: string; visitBalances: VisitBalances }) =>
-          prisonApi.getVisitBalances({ offenderNo, visitBalances }),
-        stubGetPrisoners: ({
-          results,
-          page = '0',
-          size = '10',
-        }: {
-          results: { totalPages: number; totalElements: number; content: Partial<Prisoner>[] }
-          page: string
-          size: string
-        }) => offenderSearch.getPrisoners(results, page, size),
-        stubGetPrisoner: (results: { totalPages: number; totalElements: number; content: Partial<Prisoner>[] }) =>
-          offenderSearch.getPrisoner(results),
+
+        // Prisoner Contact Registry
+        stubGetPrisonerSocialContacts: prisonerContactRegistry.stubGetPrisonerSocialContacts,
```

Comment:
Have simplified this config from what was auto-generated during upgrade from v9 to v10 (having looked at what other projects are doing...)